### PR TITLE
ref(dosDontsBlock): reuse BlockInjectButton

### PR DESCRIPTION
**Pull request by celine-s**

### CODE REVIEW

File: packages/shared/src/components/AddImagesButton.tsx [hunk 1]
```diff
@@ -2,25 +2,31 @@
...
     isLoading?: boolean;
     label: string;
     secondaryLabel?: string;
     icon?: JSX.Element;
-    onDrop: (files: FileList) => void;
+    onDrop?: (files: FileList) => void;
     fillParentContainer?: boolean;
     onUploadClick: () => void;
     onAssetChooseClick: () => void;
+    setIsMenuOpen?: (isOpen: boolean) => void;
```
Comment (fulopdaniel) [inline packages/shared/src/components/AddImagesButton.tsx:23]: maybe rename it to something more generic? `onOpenChange` ?

File: packages/dos-donts-block/src/components/AddItemButton.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { joinClassNames, textAndBgColor, textAndBgColorHover } from '@frontify/guideline-blocks-shared';
+import { AddItemButtonProps } from '../types';
+
+const AddItemButton = ({ onClick, label, icon, secondaryLabel }: AddItemButtonProps) => (
```
Comment (fulopdaniel) [inline packages/dos-donts-block/src/components/AddItemButton.tsx:6]: I think you can reuse the same component here (AddImagesButton, or if you rename it BlockInjectButton)

Comment (celine-s) [reply packages/dos-donts-block/src/components/AddItemButton.tsx:6]: I was somehow confused to reuse the other one as there is much more logic in it then in this one. It is just most of the styles it reuses. You know what I mean?

File: packages/shared/src/components/AddImagesButton.tsx (hunk 1 above)
Comment (celine-s) [reply packages/shared/src/components/AddImagesButton.tsx]: true. I removed it an used the onClick which is anyway needed for the ones where the openMenu is not necessary.

File: packages/dos-donts-block/src/components/AddItemButton.tsx (hunk 2 above)
Comment (celine-s) [reply packages/dos-donts-block/src/components/AddItemButton.tsx]: You are right. Removed it and made the other block more generic.

**Pull request merged**